utils/docker/Dockerfile

```diff
@@ -57,6 +59,10 @@ RUN curl -L https://github.com/kubernetes-sigs/kustomize/releases/download/kusto
     tar -zxf /tmp/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz -C /usr/local/bin/ && \
     rm /tmp/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz
 
+# Install helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
```

Comment:
even though we pin the version of HELM in line 14, the version is not being used here in the installation command, so you will get the latest one I presume.